### DISCUSSION TITLE
Checksum validation of migration script (up.sql/ down.sql)

### DISCUSSION THREAD

**Discussion by Leeyar** [hasura/graphql-engine#9208] [Question]
How can we make sure previous migration scripts are unaltered. Like how liquibase produces checksum and compare against the change-set checksum to make sure previous versions are unaltered and everything going incrementally.

**Comment by adas98012** [comment]
@Leeyar 
Have you considered integrating your project with Github ? Hasura supports Github integration for Cloud and Enterprise:

https://hasura.io/docs/latest/deployment/hasura-cloud/ci-cd/github-integration/#link-github-account-to-a-hasura-cloud-project

**Reply by Leeyar** [reply]
@adas98012 yes we are integrated with github.

**Reply by adas98012** [reply]
@Leeyar 
If you are integrated with github, you can then track any changes done to the migration scripts, can't you ?
